helix-gateway/src/main/java/org/apache/helix/gateway/api/service/HelixGatewayServiceChannel.java

```diff
@@ -35,10 +35,9 @@ public interface HelixGatewayServiceChannel {
    * Gateway service send a state transition message to a connected participant.
    *
    * @param instanceName the name of the participant
-   * @param currentState the current state of the shard
    * @param message      the message to send
    */
-  void sendStateTransitionMessage(String instanceName, String currentState, Message message);
+  void sendStateTransitionMessage(String instanceName,Message message);
```

Comment:
Here's the confusing part:
1. If you are talking about Helix state transition, why not still have state transition message? We will have target state, right?
2. If it is the message from Gateway manager to channel, then let's not call it message?

Reply:
updated